### ISSUE TITLE
Some strings should not be checked

### ISSUE THREAD

**Issue by fibo** [edvardchen/eslint-plugin-i18next#4]
I found this plugin really useful, there are some cases where it complains about strings that should not be checked,

for example:

    document.addEventListener('click', () => {})

or also:

    <svg id='my-icon' viewbox='0 0 32 36' />

If somebody can give me some hint where to start I can try providing a pull request.

**Comment by edvardchen** [comment]
I'm glad that this plugin is useful for you. 

Fort the user cases you mentioned:
```js
document.addEventListener('click', () => {})
```
The plugin would look at the function calls which may have string-type argument, then check if the function name is in allow lists. [Code is here](https://github.com/edvardchen/eslint-plugin-i18next/blob/master/lib/rules/no-literal-string.js#L195)

```html
<svg id='my-icon' viewbox='0 0 32 36' />
```
Same logic, but for attribute name. [Code here](https://github.com/edvardchen/eslint-plugin-i18next/blob/ba93f1fdc554ce7c9ea530ee285ad77f85c1d014/lib/rules/no-literal-string.js#L137)

It should be easy to extend. Welcome PR, or I will work on it later?

**Comment by fibo** [comment]
@edvardchen can you please publish v2.3.1 to the registry?

**Comment by edvardchen** [comment]
oh i think i had published. I am on holiday. I will publish when I come back.

Before that, you can install the version with git url as a walk-around

**Comment by edvardchen** [comment]
release v2.4.0 @fibo

**Comment by fibo** [comment]
Bravo!

@edvardchen I found other few issues, for example autoComplete prop in input, will propose another PR in the near future.
